src/jimgw/prior.py

```diff
@@ -270,10 +272,11 @@ def __init__(self, parameter_names: list[str]):
         self.parameter_names = parameter_names
         assert self.n_dim == 1, "SinePrior needs to be 1D distributions"
         super().__init__(
-            UniformPrior(-1.0, 1.0, [f"cos_{self.parameter_names[0]}"]),
+            CosinePrior([f"{self.parameter_names[0]}"]),
```

Comment:
Why is this Cosine here

Reply:
NVM I see the offset by pi /2. Can you add a line on this is for bypassing gradient errors?